### PR TITLE
Add Demo GA Blog

### PR DESCRIPTION
**Pull request by austinlparker**
For publication 10/24

### CODE REVIEW

File: content/en/blog/2022/announcing-opentelemetry-demo-release.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+say that we’ve done just that.
+Every OpenTelemetry language SDK is
+[represented](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/service_table.md)
+in this release, with the exception of Swift -- yes, even PHP. 
+We’ve built complete [tracing flows](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/trace_service_features.md) that demonstrate a breadth of common instrumentation tasks such as:
+
+- Enriching spans from automatic instrumentation.
+- Creating custom spans for richer, more useful traces.
+- Propagating trace context automatically and manually.
+- Handling observability baggage.
```
Comment (cartermp) [inline content/en/blog/2022/announcing-opentelemetry-demo-release.md:28]: For what purpose is baggage used here (for readers who may not totally get the concept)

File: content/en/blog/2022/announcing-opentelemetry-demo-release.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+We’ve also integrated OpenTelemetry Metrics across [several services](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/metric_service_features.md) to capture runtime and business metric use cases.
+
+Now, it’d be enough to just provide a great demonstration of OpenTelemetry, but
+one thing we wanted to focus on for our 1.0 release was showing not just the
+‘how’, but the ‘why’, of OpenTelemetry.
+To that end, we’ve built a framework for implementing [failure
+scenarios](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/README.md#scenarios)
+gated by feature flags.
+In addition, we include pre-configured dashboards and walk-thrus on how to read and interpret the telemetry data each service emits to discover the underlying cause of performance regressions in the application.
```
Comment (cartermp) [inline content/en/blog/2022/announcing-opentelemetry-demo-release.md:39]: Where are the dashboards?

**Review by cartermp — APPROVED** [review]
Some minor things, looks good to me though

File: content/en/blog/2022/announcing-opentelemetry-demo-release.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+title: OpenTelemetry Demo now Generally Available!
+linkTitle: Demo GA Release
+date: 2022-10-24
+author: Austin Parker
+---
+
+Earlier this year, we announced a project to build an [OpenTelemetry
+Demo](/blog/2022/demo-announcement/), representing the
+breadth of OpenTelemetry features and languages.
+Today, the Demo SIG is proud to announce [OpenTelemetry Demo
```
Comment (svrnm) [inline content/en/blog/2022/announcing-opentelemetry-demo-release.md:11]: Want to link to the repo, SIG notes, slack channel when using "Demo SIG"?

File: content/en/blog/2022/announcing-opentelemetry-demo-release.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+v1.0](https://github.com/open-telemetry/opentelemetry-demo/tree/v1.0.0)!
+With this demo, you’ll be able to quickly run a complete end-to-end distributed system instrumented with 100% OpenTelemetry Traces and Metrics.
+
+![demo image](https://user-images.githubusercontent.com/47896520/196496223-6d6ea729-5bea-4a8c-a2c6-cd51cce386ae.png)
```
Comment (svrnm) [inline content/en/blog/2022/announcing-opentelemetry-demo-release.md:15]:
```suggestion
![The system architecture of the demo application represented as directed acyclic graph in Jaeger UI](https://user-images.githubusercontent.com/47896520/196496223-6d6ea729-5bea-4a8c-a2c6-cd51cce386ae.png)
```

Also put the text & image into a folder

File: content/en/blog/2022/announcing-opentelemetry-demo-release.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+However, just because we reached 1.0, that doesn’t mean we’re stopping -- this
+demo is a living artifact, one that we intend to continue to improve. In the
+coming months we plan to continue to iterate and improve coverage of metrics and
+logs as more SDKs reach maturity.
+
+We also hope to add new instrumentation scenarios and patterns by extending the functionality of the application -- queues and async processing of requests, a hosted version in order to explore the demo with zero setup, adding in support for Swift, and more.
+
+We’d love for you to take the demo for a spin and let us know what you think!
+Check out the
+[docs](https://github.com/open-telemetry/opentelemetry-demo/tree/main/docs), or
```
Comment (svrnm) [inline content/en/blog/2022/announcing-opentelemetry-demo-release.md:59]: any plans to have the docs at https://opentelemetry.io/community/demo/?

**Review by svrnm — APPROVED** [review]
sume suggestions, nothing that should stop us from publishing this quickly

File: content/en/blog/2022/announcing-opentelemetry-demo-release.md (hunk 5 above)
Comment (austinlparker) [reply content/en/blog/2022/announcing-opentelemetry-demo-release.md:59]: Eventually!

**Comment by cartermp** [comment]
@open-telemetry/blog-approvers PTAL!

(should this go unreviewed before it needs to get released then I'd say just merge it @austinlparker )

**Pull request merged**